digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala

```diff
@@ -93,22 +86,12 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
 
   def getRoadLinksByLinkIdsFromVVH(linkIds: Set[Long], newTransaction: Boolean = true, frozenTimeVVHAPIServiceEnabled: Boolean = false): Seq[RoadLink] = {
     val vvhRoadLinks = getVVHRoadlinks(linkIds, frozenTimeVVHAPIServiceEnabled)
-    if (newTransaction)
-      withDynTransaction {
-        enrichRoadLinksFromVVH(vvhRoadLinks)
-      }
-    else
-      enrichRoadLinksFromVVH(vvhRoadLinks)
+    enrichRoadLinksFromVVH(vvhRoadLinks)
   }
 
   def getSuravageRoadLinksByLinkIdsFromVVH(linkIds: Set[Long], newTransaction: Boolean = true): Seq[RoadLink] = {

```

Comment:
Do we need the "newTransaction: Boolean = true," part?

Reply:
I will remove those.